e2e/utils/pages/homepage.page.ts

```diff
@@ -243,6 +243,13 @@ abstract class Homepage {
 		await this.clickByTestId(NAVIGATION_ITEM_TOKENS);
 	}
 
+	async takeScreenshot(): Promise<void> {
+		await expect(this.#page).toHaveScreenshot({
+			fullPage: true,
+			timeout: 5 * 60 * 1000
```

Comment:
It's a pretty long timeout, we need it to be that long?

Reply:
Currently our standard timeout for tests is set on 5 Minutes. That is because we have certain tests that do actually need 2-3 minutes sometimes. 

This timeout here only tells playwright that it has 5 minutes to try and get the correct screenshot. So if the first one fails, it will try again. I can lower it but does not hurt at all.

Reply:
If it's standardized, should we use a constant or some constants instead of "hardcoding" the value here then?

Reply:
Well my initial idea was setting this option in the playwright config aswell. Somehow that is not supported, so the only other option I see is hardcoding it here. Basically its already reduced usage from mentioning it in every test file to now having it only on homepage.

Reply:
Should we create an environment variable for the timeout that way we can use the same value in both configuration and code? (obviously if yes in another PR)

Reply:
I feel like a constant would be good enough for this case. But then again I wonder if its not a bit too much just for a timeout. Lets say in a few weeks we want to change the timeout in the config. Then we would need to create two constants for both cases or adjust the one in the test too.

Reply:
Works for me. Let’s see how it evolves. I just fear that we might end up with multiple hardcoded timeouts, which could contradict each other at some point if we don’t pay attention.